### ISSUE TITLE
REST_edges

### ISSUE THREAD

**Issue by touch4564** [hvppycoding/runrest#1]
Is it possible to get the edge of the RSMT from this algorithm?

**Comment by hvppycoding** [comment]
Yes, it can generate edges from the points; the number of points is limited to 50 or fewer because there are no trained models for more than 50 points.
The output might require additional post-processing (e.g., Steiner point generation) if you use it in a global router.
It is just a wrapper for REST; you can refer to the original repository(https://github.com/cuhk-eda/REST) if you want to know the detailed algorithm.

**Comment by touch4564** [comment]
How to generate Steiner points, is it modified in this function（plot_rest)?

**Comment by hvppycoding** [comment]
Steiner point generation is not included in the REST project. Instead, it just evaluates the total length by removing duplicate segments using the `Evaluator` class.

**Comment by touch4564** [comment]
It still feels like the code to generate the Steiner point is still difficult, do you have any suggestions? I'm really sorry, the main thing is that my code skills are very average, and I mainly want to use the algorithm to split into multiple two-pin wiring

**Comment by hvppycoding** [comment]
I agree that Steinerizing REST results is not straightforward. I've implemented this feature in another project as well. It involves adding Steiner points by merging two edges that share a segment. Apologies for the untidy code.

https://github.com/hvppycoding/allrest/blob/main/src/allrest/treeconverter.py#L404

**Comment by touch4564** [comment]
Thank you for your reply！
